ntoskrnl/ntoskrnl.spec

```diff
@@ -1228,7 +1228,7 @@
 @ stdcall RtlLookupElementGenericTableAvl(ptr ptr)
 @ stdcall RtlLookupElementGenericTableFull(ptr ptr ptr ptr)
 @ stdcall RtlLookupElementGenericTableFullAvl(ptr ptr ptr ptr)
-@ cdecl -arch=x86_64,arm RtlLookupFunctionEntry(double ptr ptr)
+@ cdecl -arch=x86_64 RtlLookupFunctionEntry(double ptr ptr)
```

Comment:
This looks like a correction / partial revert of the previous commit. They should probably be squashed together.